File: GFG/Array/Day2/RotateArray.java
public class RotateArray{
    public static void rotateArr(int arr[], int d) {
        // code here
        int res[]=new int[arr.length];
        
        for(int i=0;i<arr.length;i++){
            res[i]=arr[(i+d)%arr.length];
        }
        
        for(int i=0;i<arr.length;i++){
            arr[i]=res[i];
        }
    }
}

File: GFG/Array/Day2/RotateArray.py
class Solution:
    def rotateArr(self, arr, d):
        #code here
        res=[0]*len(arr)
        
        for i in range(len(arr)):
            res[i]=arr[(i+d)%len(arr)]
            
        for i in range(len(arr)):
            arr[i]=res[i]